README.md

```diff
@@ -1 +1,30 @@
-# camp2-helpers
+# Camp2 Helpers
+
+This package modifies the jest [expect](https://jestjs.io/docs/en/expect) function and wrap it in a custom package in order to give an extra option to add a message when jest tests fails in Typescript.
+It is based the package [jest-expect-message](https://www.npmjs.com/package/jest-expect-message) for custom use.
+
+## Usage
+
+### Installation
+
+To install it, add this in your `package.json` file:
+
+```json
+"dependencies": {
+  "camp2-helpers": "fewlines-education/camp2-helpers#master"
+}
+```
+
+### Import
+
+### Utility functions
+
+#### readline
+
+#### nodeParser
+
+#### expectMessage
```

Comment:
If someone want to use the package he will need to hard try it or read the code to understand how to use it.
You resolved the **[comment](https://github.com/fewlines-education/camp2-helpers/pull/1/files#r446905241)** where I asked for code example, could you add at least one usage example? 🙂

Reply:
Ho my bad, I thought we made it clear that modifications in the README would be done in a second time when all the utility functions are implemented! (it's actually WIP in the second PR of this repo !)

Reply:
👍